frontend/src/concepts/areas/AreaContext.tsx

```diff
@@ -34,11 +36,22 @@ const AreaContextProvider: React.FC<AreaContextProps> = ({ children }) => {
   const loaded = loadedDsc && loadedDsci;
 
   const contextValue = React.useMemo(() => ({ dscStatus, dsciStatus }), [dscStatus, dsciStatus]);
-  if (error) {
+
+  if (error || (dscStatus && Object.keys(dscStatus).length === 0)) {
     return (
-      <Alert isInline variant="danger" title="Problem loading component state">
-        {error.message}
-      </Alert>
+      <Page>
+        <ApplicationsPage
+          loaded
+          empty={false}
+          loadError={error}
+          loadErrorPage={
+            <RedirectErrorState
+              title="Could not load component state"
+              errorMessage={error?.message}
+            />
+          }
+        />
+      </Page>
```

Comment:
In the case where the dscStatus is empty/missing but there is no `loadError` here, the `loadErrorPage` doesn't render and you get a blank page. Since this entire return statement is for the error/missing case and we don't need to conditionally show or not show an error, I don't think we need to use the `loadError`/`loadErrorPage` functionality of ApplicationsPage. Instead we can just pass the RedirectErrorState to it as children:

```tsx
<ApplicationsPage loaded empty={false}>
  <RedirectErrorState
    title="Could not load component state"
    errorMessage={error?.message}
  />
</ApplicationsPage>
```